velox/functions/lib/SubscriptUtil.h

```diff
@@ -23,6 +23,49 @@
 
 namespace facebook::velox::functions {
 
+namespace {
```

Comment:
@oerling We should not have anonymous namespace in header file.  Maybe using `detail` namespace instead